src/helpers/formRenderer.js

```diff
@@ -2,7 +2,8 @@ import { Container } from 'components/Container.jsx';
 import React from 'react';
 import ReactDOM from 'react-dom';
 import ControlRecordTreeBuilder from 'src/helpers/ControlRecordTreeBuilder';
-import ScriptRunner from './scriptRunner';
+import ScriptRunner from './ScriptRunner';
```

Comment:
Why have u chnaged this to captial S?? "ScriptRunner"?

Reply:
Normally we changed, i can revert it back to small s.